src/sidebar/services/test/toast-messenger-test.js

```diff
@@ -0,0 +1,157 @@
+import toastMessenger from '../toast-messenger';
+
+describe('toastMessenger', function() {
```

Comment:
Honestly, I forgot our standard for this, and perhaps we need to resist in the next front end sync. For services with multiple exports, should we have this the full file path or just the server file name or the service name?
"sidebar/services/toastMessenger" for example? I think in another PR, we may do a clean up of all test files in services/  I'm happy to take that on.